content/zh/docs/setup/production-environment/turnkey/gce.md

```diff
@@ -0,0 +1,397 @@
+-->
+如果你希望在项目中运行多个集群，希望使用一个不同名称，或者不同数量工作节点的集群，
+请查看 `<kubernetes>/cluster/gce/config-default.sh` 文件，以便在启动集群之前进行更细粒度的配置。
+
+
+<!--  
+If you run into trouble, please see the section on [troubleshooting](/docs/setup/production-environment/turnkey/gce/#troubleshooting), post to the
+[Kubernetes Forum](https://discuss.kubernetes.io), or come ask questions on `#gke` Slack channel.
+-->
+如果你遇到了问题，请参阅[错误排查](/zh/docs/setup/production-environment/turnkey/gce/#troubleshooting)一节，
+发布到 [Kubernetes 论坛](https://discuss.kubernetes.io)，或者来 `#gke` Slack 频道中提问。
+
+<!-- The next few steps will show you: -->
+接下来的几个步骤会告诉你:
+
+<!-- 
+1. How to set up the command line client on your workstation to manage the cluster
+2. Examples of how to use the cluster
+3. How to delete the cluster
+4. How to start clusters with non-default options (like larger clusters) 
+-->
+1. 如何在你的工作站设置命令行客户端来管理集群
+2. 如果使用集群的示例
+3. 如何删除集群
+4. 如果以非默认选项启动集群（如规模较大的集群）
+
+<!-- ## Installing the Kubernetes command line tools on your workstation -->
+## 在你的工作站安装 Kubernetes 命令行工具
+
+<!--  
+The cluster startup script will leave you with a running cluster and a `kubernetes` directory on your workstation.
+-->
+集群启动脚本将在你的工作站上留下一个正在运行的集群和一个 `kubernetes` 目录。
+
+<!--  
+The [kubectl](/docs/reference/kubectl/kubectl/) tool controls the Kubernetes cluster
+manager.  It lets you inspect your cluster resources, create, delete, and update
+components, and much more. You will use it to look at your new cluster and bring
+up example apps.
+-->
+[kubectl](/zh/docs/reference/kubectl/kubectl/) 工具控制 Kubernetes 集群管理器。
+它允许你检查集群资源，创建、删除和更新组件等等。
+你将使用它来查看新集群并打开示例应用程序。
+
+<!--  
+You can use `gcloud` to install the `kubectl` command-line tool on your workstation:
+-->
+你可以使用 `gcloud` 在工作站上安装 `kubectl` 命令行工具:
+
+```shell
+gcloud components install kubectl
+```
+
+{{< note >}}
+<!--  
+The kubectl version bundled with `gcloud` may be older than the one
+downloaded by the get.k8s.io install script. See [Installing kubectl](/docs/tasks/tools/install-kubectl/)
+document to see how you can set up the latest `kubectl` on your workstation.
+-->
+与 `gcloud` 绑定的 kubectl 版本可能比 get.k8s.io 下载的更老。
+查看[安装 kubectl](/zh/docs/tasks/tools/install-kubectl/) 文档，了解如何在工作站上设置最新的 `kubectl`。
+{{< /note >}}
+
+<!-- ## Getting started with your cluster -->
+## 开始使用你的集群
+
+<!-- ### Inspect your cluster -->
+### 检查你的集群
+
+<!--  
+Once `kubectl` is in your path, you can use it to look at your cluster. E.g., running:
+-->
+一旦 `kubectl` 存在于你的路径中，你就可以使用它来查看集群，例如，运行：
+
+```shell
+kubectl get --all-namespaces services
+```
+
+<!--  
+should show a set of [services](/docs/concepts/services-networking/service/) that look something like this:
+-->
+应该显示 [services](/zh/docs/concepts/services-networking/service/) 集合，看起来像这样：
+
+```shell
```

Comment:
this is not shell

Reply:
why it is not ?

Reply:
When you apply 'shell' to a code block, the block will be syntax highlighted as *Shell Commands*.
We do not highlight shell outputs as shell commands because the highlighting might be confusing.

Reply:
so it is. shell command should not be same as shell output